File: 8KYU/Find the Slope/solution.clj
(ns Slope)  

(defn slope [v]
    (let [delta-x (- (nth v 3) (second v))
          delta-y (- (nth v 2) (first v))]
      (if (= 0 delta-y)
        "undefined"
        (str (/ delta-x delta-y)))))

File: 8KYU/Find the Slope/solution.cs
using System;
public class Slope
{

  public String slope(int[] points)
  {
    var deltaX = points[3] - points[1];
    var deltaY = points[2] - points[0];
    
    if(deltaY == 0)
      return "undefined";
    return (deltaX / deltaY).ToString();
  }

}